clang/test/SemaCXX/cheri/cheri-dependent-convert.cpp

```diff
@@ -0,0 +1,29 @@
+// RUN: %cheri_cc1 -std=c++17 -fsyntax-only %s -verify=expected,hybrid
```

Comment:
```suggestion
/// Check that we emit correct diagnostics for capability <-> pointer/int mismatches
/// in template parameter lists.
// RUN: %cheri_cc1 -std=c++17 -fsyntax-only %s -verify=expected,hybrid
```
Maybe add something like this ?

Reply:
Well, it's everything, both that we emit diagnostics in the instantiations and that we don't emit bogus errors for the template definition itself due to having a dependent type.

Reply:
Could you include that in a comment? I like having a comment at the top of each test that briefly summarizes what is being tested.